miden-lib/asm/sat/account.masm

```diff
@@ -218,3 +221,43 @@ export.set_code
     exec.layout::set_new_acct_code_root
     # => []
 end
+
+#! Gets an item from the account storage. Panics if the index is out of bounds.
+#!
+#! Stack: [index]
+#! Output: [VALUE]
+#!
+#! - index is the index of the item to get.
+#! - VALUE is the value of the item.
+export.get_item
+    # get the storage root
+    exec.layout::get_acct_storage_root
+    # => [storage_root, index]
+
+    # get the item from storage
+    movup.4 push.STORAGE_TREE_DEPTH mtree_get
+    # => [VALUE, ROOT]
```

Comment:
Should there be a `swapw dropw` here to make sure we drop the root?

Reply:
added